x-pack/plugins/code/server/__tests__/lsp_service.ts

```diff
@@ -228,4 +217,37 @@ describe('lsp_service tests', () => {
     }
     // @ts-ignore
   }).timeout(10000);
+
+  it('should work if a worktree exists', async () => {
```

Comment:
👍 for this test. I notice that for `workspace_handler`, `clone_workspace` and `update_workspace` do not have unit tests yet, are you trying to cover them in this `lsp_service` unit test?

Reply:
I think clone_workspace is covered here but update_workspace is not. We might add one.

Reply:
done.